01/ht/DashPisareva/03-table/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Генератор таблицы</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <form>
+      Выбор класса:
+        <input id="class" size="15" oninput="setClass(this.value)">turq/bl/yel</input>
+      </br>Выбор строк:
+        <select id="rowNum"></select></br>
+      Выбор колонок:
+        <select id="colNum"></select></br>
+      <input type="checkbox" id="enableHeader" />
+      <label>Head</label></br>
+        <input type="checkbox" id="enableCaption" />
+        <label>Caption</label>
+  </form>
+  </br>
+  <button class="btn" id="makeTable" onclick="generateTable()">Генерировать таблицу</button>
+    <div id="fullTable">
+      </br>
+    <div id="header"></div>
+
+<!--Таблица-->
+<div id="tableDiv" onclick="changeHtml()"></div>
```

Comment:
don't use such handlers. It might be ok for this task, but in general it's bad idea